lib/charms/tempo_k8s/v1/charm_tracing.py

```diff
@@ -15,14 +15,14 @@
 `@trace_charm(tracing_endpoint="my_tracing_endpoint")`
 
 2) add to your charm a "my_tracing_endpoint" (you can name this attribute whatever you like) **property**
-that returns an otlp grpc endpoint url. If you are using the `TracingEndpointProvider` as
+that returns an otlp http endpoint url. If you are using the `TracingEndpointProvider` as
 `self.tracing = TracingEndpointProvider(self)`, the implementation could be:
 
 ```
     @property
     def my_tracing_endpoint(self) -> Optional[str]:
         '''Tempo endpoint for charm tracing'''
-        return self.tracing.otlp_grpc_endpoint
+        return f"http://{self.tracing.otlp_http_endpoint}/v1/traces"
```

Comment:
shouldn't this URLification be done by the tracing lib?

Reply:
I changed the approach as discussed. Now:
- `tracing` lib adds the protocol part if it's needed based on whether we're using one of the HTTP protocols and SSL is enabled or not
- `charm_tracing` lib, as it uses `otlp/http`, adds the `/v1/traces` suffix to the URLs so `tracing` lib can stay generic for other instrumentation libraries

There's a potential of further improvements to remove the need to specify the endpoint altogether in future but I'll create a separate enhancement issue for that. 

Please have a look.